libs/sdk-ui-dashboard/src/plugins/customizer.ts

```diff
@@ -499,6 +556,11 @@ export interface IDashboardCustomizer {
      * Customize how rendering of filters is done.
      */
     filters(): IFiltersCustomizer;
+
+    /**
+     * Customize dashboard content.
+     */
+    dashboardContent(): IDashboardContentCustomizer;
```

Comment:
I would keep this just "dashboard" so we can extend it later if needed.

Reply:
Renamed.